src/main/java/racingcar/model/Car.java

```diff
@@ -0,0 +1,33 @@
+package racingcar.model;
+
+import static racingcar.constants.Symbol.COLON;
+import static racingcar.constants.Symbol.HYPHEN;
+import static racingcar.constants.Symbol.SPACE;
+import static racingcar.constants.Symbol.ZERO;
+
+public class Car {
+    private final CarName carName;
+    private Integer position;
+
+    public Car(CarName carName) {
+        this.carName = carName;
+        this.position = ZERO;
+    }

```

Comment:
p4

`position`은 항상 0으로 초기화가 되기 때문에  인스턴스 변수에서 바로 초기화하는 건 어떠신지요?
만약 인자를 받아 초기화하거나, 다른 변수나 객체의 상태에 따라 값이 달라지지 않기 때문에  한 번 고려해보시는 것도 좋을 것 같습니다.

저같은 경우 **고정된 초기값**이 필요하다면 **인스턴스 변수에서 초기화**를 하고 **동적 초기값**이 필요하거나 다양한 초기화 로직이 필요하다면 **생성자에서 초기화**하고 있습니다. 😊

```suggestion
    private final CarName carName;
    private Integer position = ZERO;

    public Car(CarName carName) {
        this.carName = carName;
    }

```

Reply:
헉.. 그러네요 사소한 부분인데 이러한 부분까지 봐주시다니.. 제 코드 리뷰에 얼마나 신경을 많이 써주셨는지가 느껴집니다... 정말 감사합니다..!!

인스턴스 변수에서 초기화하는 것이랑 생성자에서 초기화하는 것에 대해서는 깊게 생각해보지 못했는데,
철원님 덕분에 생각 확장의 계기가 되네요..!!

Car 객체가 아무래도 Car들을 개별단위로 다루다 보니, 인스턴스 변수에서 초기화하는 것이 더 좋아보이네요!!